### DISCUSSION TITLE
Implement of mediastream on addTrack of peerConnection

### DISCUSSION THREAD

**Discussion by kolserdav** [shinyoshiaki/rainy-webrtc#239] [Q&A] [answered]
Hello! You are planning to implement this https://github.com/shinyoshiaki/werift-webrtc/blob/develop/packages/webrtc/src/peerConnection.ts#L1089 ?

**Comment by shinyoshiaki** [comment, marked as answer]
I don't have any immediate plans to do it, but I will someday.
